Adds a mission where the player is invited to a Coalition university seminar

roadrunner56 opened the pull request:
**Content (Artwork / Missions / Jobs)**

## Summary
It seemed strange to me that as the player travels through Coalition space, few NPCs show much interest towards the player and their knowledge of the broader galaxy. I thought that there would probably be at least some Coalition citizens who would want to hear about the player's adventures and learn more about humanity. This mission has the player invited to a xenology seminar at Starlit University, where they have the option of telling the attendees about humanity and the galaxy as a whole. 
I realize that this may have broader implications for future storylines/missions involving the Coalition, so I avoided talking about the Quarg and Pug. Even so, if this seems too radical of an idea, then please say so.
I'm awful at making mission syntax work, so this mission is probably super buggy. Apologies in advance. Once again, thank you to all of the people on Discord for helping me make this mission work.

## Save File
Any save file where the player has access to a jump drive and has done the Coalition first contact missions will work. Be advised that the conversation changes based on whether or not the player has met the Hai, Korath, and has finished FW Reconciliation/Checkmate.

Pointedstick commented on data/coalition/coalition missions.txt:
```diff
@@ -1684,3 +1684,180 @@ mission "Coalition: Alpha Encounter"
+
+
+mission "Saryd University Lecture"
+	name "Saryd Scientists"
```
Needs a verb; maybe "transport?"

```suggestion
	name "Transport Saryd Scientists"
```

Pointedstick commented on data/coalition/coalition missions.txt:
```diff
@@ -1684,3 +1684,180 @@ mission "Coalition: Alpha Encounter"
+	destination "Shadow of Leaves"
+	on offer
+		conversation
+			`As you exit from a restaurant in the spaceport, you notice a rumpled-looking Saryd staring at you. This is not particularly unusual for you, as you are quite a strange sight here in Coalition space. However, he then begins to gallop towards you, hailing you down.`
```
```suggestion
			`As you exit a restaurant in the spaceport, you notice a rumpled-looking Saryd staring at you. This is not particularly unusual, as consider what a strange sight you must be here in Coalition space. However, he then begins to gallop towards you, hailing you down.`
```

Pointedstick commented on data/coalition/coalition missions.txt:
```diff
@@ -1684,3 +1684,180 @@ mission "Coalition: Alpha Encounter"
+		"coalition jobs" += 3
+		payment 232000
+		conversation
+			`During the trip, Aelbris was polite, if a little excited to have a human as his pilot. He asked you several questions during the flight, which you answered as best as you can. Once you and your passenger departs from your ship, Aelbris turns around.`
```
```suggestion
			`During the trip, Aelbris was polite, if a little excited to have a human as his pilot. He asked many questions during the flight, which you answered as best you could. Once you and your passenger departs from your ship, Aelbris turns around.`
```

Pointedstick commented on data/coalition/coalition missions.txt:
```diff
@@ -1684,3 +1684,180 @@ mission "Coalition: Alpha Encounter"
+					accept
+	on complete
+		"coalition jobs" += 3
+		payment 232000
```
Sort of an oddly specific non-round number. Why not round it up or down, or else use a modification of the standard payment calculation?

roadrunner56 replied:
I changed it to be a bit less, as upon thinking about it 232000 seems a bit much for one bunk and a couple of jumps.

Pointedstick commented on data/coalition/coalition missions.txt:
```diff
@@ -1684,3 +1684,180 @@ mission "Coalition: Alpha Encounter"
+		payment 232000
+		conversation
+			`During the trip, Aelbris was polite, if a little excited to have a human as his pilot. He asked you several questions during the flight, which you answered as best as you can. Once you and your passenger departs from your ship, Aelbris turns around.`
+			`	"Captain, that you must be very busy, I realize, but wondering if you could stay for the seminar, I was. You see, studying human space for years, I have been, but actually meet a human, I have not. A great boon to our field to have a human speak at our seminar, it would be."`
```
```suggestion
			`	"Captain, that you must be very busy, I realize, but wondering if you could stay for the seminar, I was. You see, studying human space for years, I have been, but actually meet a human, I had not. A great boon to our field to have a human guest speaker, it would be."`
```

W1zrad replied:
```suggestion
			`	"Captain, that you must be very busy, I realize, but wondering if you could stay for the seminar, I was. You see, studying human space for years, I have been, but actually met a human, I had not. A great boon to our field to have a human guest speaker, it would be."`
```
Just adding a small change to Pointed's suggestion.

Pointedstick commented on data/coalition/coalition missions.txt:
```diff
@@ -1684,3 +1684,180 @@ mission "Coalition: Alpha Encounter"
+				`	"Sure, I'd be glad to."`
+					goto "yes"
+				`	"Sorry, I don't have time."`
+						decline
```
```suggestion
					decline
```

Pointedstick commented on data/coalition/coalition missions.txt:
```diff
@@ -1684,3 +1684,180 @@ mission "Coalition: Alpha Encounter"
+				`	"Sure, I'd be glad to."`
+					goto "yes"
+				`	"Sorry, I don't have time."`
+						decline
```
```suggestion
					decline
```

Pointedstick commented on data/coalition/coalition missions.txt:
```diff
@@ -1684,3 +1684,180 @@ mission "Coalition: Alpha Encounter"
+				`	"Sure, I'd be glad to."`
+					goto "yes"
+				`	"Sorry, I don't have time."`
+						decline
```
```suggestion
					decline
```

Pointedstick commented on data/coalition/coalition missions.txt:
```diff
@@ -1684,3 +1684,180 @@ mission "Coalition: Alpha Encounter"
+				`	"Sorry, I don't have time."`
+						decline
+			label "explanation"
+			`	"Close to Arachi space, humans are, so study radio and TV signals from your part of the galaxy, we are able to. Faint, scattered broadcasts, sophisticated receivers collect, and the signals to us, Arachi colleagues relay. From these broadcasts, we learn. Help us learn more, would you?"`
```
```suggestion
			`	"Close to Arachi space, humans are, so study radio and television signals from your part of the galaxy, we are able to. Faint, scattered broadcasts, sophisticated receivers collect, and the signals to us, Arachi colleagues relay. From these broadcasts, we learn. Help us learn more, would you?"`
```

Pointedstick commented on data/coalition/coalition missions.txt:
```diff
@@ -1684,3 +1684,180 @@ mission "Coalition: Alpha Encounter"
+				`	"Sure, I'd be glad to."`
+					goto "yes"
+				`	"Sorry, I don't have time."`
+						decline
```
```suggestion
					decline
```

Pointedstick commented on data/coalition/coalition missions.txt:
```diff
@@ -1684,3 +1684,180 @@ mission "Coalition: Alpha Encounter"
+						decline
+			label "yes"
+			`	"Thank you! A great honor, you have done our field of study." He motions you to follow him.`
+			`	The train you took to the university arrives remarkably quickly, and you are on your way towards a large building.`
```
```suggestion
			`	The train you took to the university arrives remarkably quickly, and you are soon on your way towards a large building.`
```

Pointedstick commented on data/coalition/coalition missions.txt:
```diff
@@ -1684,3 +1684,180 @@ mission "Coalition: Alpha Encounter"
+			label "yes"
+			`	"Thank you! A great honor, you have done our field of study." He motions you to follow him.`
+			`	The train you took to the university arrives remarkably quickly, and you are on your way towards a large building.`
+			`	You enter a room that is filled with tables of citizens from across the Coalition. A stage is at the front, where a Kimek begins to set up some sort of model. Your entrance immediately causes a stir, and several turn to look at you. Aelbris beams as he shows you to your table."`
```
```suggestion
			`	You enter a room full of tables packed with citizens from across the Coalition. A stage is at the front, where a Kimek is setting up some sort of diorama. Your entrance immediately causes a stir, and several turn to look at you. Aelbris beams as he shows you to your table."`
```

Pointedstick commented on data/coalition/coalition missions.txt:
```diff
@@ -1684,3 +1684,180 @@ mission "Coalition: Alpha Encounter"
+			`	You enter a room that is filled with tables of citizens from across the Coalition. A stage is at the front, where a Kimek begins to set up some sort of model. Your entrance immediately causes a stir, and several turn to look at you. Aelbris beams as he shows you to your table."`
+			`	"Now, unplanned, your visit is, so consult with the seminar organizers, I must. Be right back, I will."`
+			`	You wait for a while, until an Arach walks up to the stage. The box's volume is much louder than normal, and you can easily hear what is being spoken.`
+			`	"Pleased to announce that the 103rd Xenology Seminar, generously hosted by Starlit University, is now in session, I am. Noticed as you may have, in the presence of a very... unusual guest, we are. The honor of having <first> <last>, of the Rutilicus system, answer some questions at the end of today's seminar, we will have." The Arach gestures to you, as Aelbris sits down at your table.`
```
```suggestion
			`	"Pleased to announce that the 103rd Xenology Seminar, generously hosted by Starlit University, is now in session, I am. Noticed as you may have, in the presence of a very... unusual guest, we are. The honor of having <first> <last> of the Rutilicus system answer some questions at the end of today's seminar, we will have." The Arach gestures to you, as Aelbris sits down at your table.`
```

Pointedstick commented on data/coalition/coalition missions.txt:
```diff
@@ -1684,3 +1684,180 @@ mission "Coalition: Alpha Encounter"
+			`	"Now, unplanned, your visit is, so consult with the seminar organizers, I must. Be right back, I will."`
+			`	You wait for a while, until an Arach walks up to the stage. The box's volume is much louder than normal, and you can easily hear what is being spoken.`
+			`	"Pleased to announce that the 103rd Xenology Seminar, generously hosted by Starlit University, is now in session, I am. Noticed as you may have, in the presence of a very... unusual guest, we are. The honor of having <first> <last>, of the Rutilicus system, answer some questions at the end of today's seminar, we will have." The Arach gestures to you, as Aelbris sits down at your table.`
+			`	The first speaker, a Kimek, talks about human farming methods as they compare to other members of the Coalition. Strangely enough, you see him cite human news broadcasts and television shows as if they were academic papers. All of the broadcasts are hundreds of years out of date. The whole thing feels surreal, as if you were viewing your own society from the outside.`
```
```suggestion
			`	The first speaker, a Kimek, talks about human farming methods as they compare to other members of the Coalition. Strangely enough, you see him cite human news broadcasts and television shows as if they were academic papers. All of the broadcasts are hundreds of years out of date. Viewing your own society from the outside feels surreal.`
```

Pointedstick commented on data/coalition/coalition missions.txt:
```diff
@@ -1684,3 +1684,180 @@ mission "Coalition: Alpha Encounter"
+				`	"No, the only aliens humans know about are the Quarg."`
+					goto "sad"
+			label "sad"
+			`	At this several in the audience sound disappointed.`
```
```suggestion
			`	At this, several members of the audience elicit disappointed noises`
```

W1zrad replied:
```suggestion
			`	At this, several members of the audience elicit disappointed noises.`
```

W1zrad replied:
That's a fair point!
```suggestion
			`	At this, several disappointed noises are elicited from the audience.`
```
Thoughts? Seems overly fancy but it's all that popped into my head.

Pointedstick commented on data/coalition/coalition missions.txt:
```diff
@@ -1684,3 +1684,180 @@ mission "Coalition: Alpha Encounter"
+			`	At this several in the audience sound disappointed.`
+				goto "nextquestion"
+			label "know"
+			`	At this several in the audience sound delighted. "Cooperation between different species, it is important to foster," says the Saryd who asked the question. An Arach asks you to point out where the Hai are on a galactic map, and you do so.`
```
```suggestion
			`	Several members of the audience are delighted. "Cooperation between different species, it is important to foster," says the Saryd who asked the question. An Arach asks you to point out where the Hai are on a galactic map, and you do so.`
```

Pointedstick commented on data/coalition/coalition missions.txt:
```diff
@@ -1684,3 +1684,180 @@ mission "Coalition: Alpha Encounter"
+			`	The Arach selects another, this time a Kimek. "All of human space, united politically is it?"`
+			choice
+				`	"No, human space is split between the Republic in the north and the Free Worlds in the south."`
+				`	"No, human space is split between the Republic in the west, the Free Worlds in the south, and the Syndicate in the east. The Syndicate is technically part of the Republic, but they're very autonomous."`
```
```suggestion
				`	"No, human space is split between the Republic in the west, the Free Worlds in the south, and the Syndicate in the east. The Syndicate is technically part of the Republic, but they have a lot of autonomy."`
```

Pointedstick commented on data/coalition/coalition missions.txt:
```diff
@@ -1684,3 +1684,180 @@ mission "Coalition: Alpha Encounter"
+			`	"Sorry, I don't know what's there any more than you do."`
+			`	The Saryd walks away, disappointed.`
+			label "tell"
+			`	"An alien species known as the Korath live there. They're reptillian aliens whose space has been torn apart by fighting. There are four factions. The Kor Sestor and Kor Mereti have been fighting a civil war using solely autonomous ships. The Kor Efreti have stayed out of the civil war, living under the Quarg's protection." At this there are hisses from several in the crowd. "The last faction ...
```
```suggestion
			`	"An alien species known as the Korath live there. They're reptillian creatures whose space has been torn apart by fighting. There are four factions. The Kor Sestor and Kor Mereti have been fighting a civil war using autonomous robotic ships. The Kor Efreti have stayed out of the civil war, living under the Quarg's protection." At this there are hisses from several in the crowd. "The last faction, the Korath Exiles, have been banished to the Core by the Quarg, for refusing to give up their ability to manufacture jump drives. They are now raiding humanity for supplies."`
```

Pointedstick commented on data/coalition/coalition missions.txt:
```diff
@@ -1684,3 +1684,180 @@ mission "Coalition: Alpha Encounter"
+			`	"Driven to such desperation, it is unfortunate that our allies against the Quarg are. Powerful like the Coalition, hopefully one day they will be."`
+				goto "exit"
+			label "sorry"
+			`	The crowd's mood dampens a bit, but is still exuberant as you leave the building. It is now night, and your eyes take a moment to adjust from the lights of the lecture hall to the dark.`
```
```suggestion
			`	The crowd's mood dampens a bit, but everyone is still quite exuberant as you leave the building. It is now night, and your eyes take a moment to adjust from the lecture hall's bright lights.`
```

Pointedstick commented on data/coalition/coalition missions.txt:
```diff
@@ -1684,3 +1684,180 @@ mission "Coalition: Alpha Encounter"
+			`	The crowd's mood dampens a bit, but is still exuberant as you leave the building. It is now night, and your eyes take a moment to adjust from the lights of the lecture hall to the dark.`
+				decline
+			label "exit"
+			`	The crowd's mood is exuberant as you leave the building. It is now night, and your eyes take a moment to adjust from the lights of the lecture hall to the dark.`
```
```suggestion
			`	The crowd's mood is exuberant as you leave the building. It is now night, and your eyes take a moment to adjust from the lecture hall's bright lights.`
```

Pointedstick reviewed (changes requested):
Great stuff!

W1zrad commented on data/coalition/coalition missions.txt:
```diff
@@ -1684,3 +1684,180 @@ mission "Coalition: Alpha Encounter"
+		conversation
+			`As you exit from a restaurant in the spaceport, you notice a rumpled-looking Saryd staring at you. This is not particularly unusual for you, as you are quite a strange sight here in Coalition space. However, he then begins to gallop towards you, hailing you down.`
+			`	"Captain! Excited to finally meet you, I am. Heard of the human in our space, I have, but to see you myself, special it is."`
+			`	"A scientist, I am, and invitied to speak at a university seminar, I have been." This time, he speaks in a calmer tone of voice. "Be there by <date>, I must. The honor of transporting us to Shadow of Leaves, will you give me?"`
```
```suggestion
			`	"A scientist, I am, and invitied to speak at a university seminar, I have been." This time, he speaks in a calmer tone of voice. "Be there by <date>, I must. The honor of transporting us to <planet>, will you give me?"`
```

W1zrad commented on data/coalition/coalition missions.txt:
```diff
@@ -1684,3 +1684,180 @@ mission "Coalition: Alpha Encounter"
+				`	"Sorry, I can't take transport jobs right now."`
+					decline
+			label "xeno"
+			`	"Funny you ask, it is. Xenologist, I am. Alien culture and physiology, I study, and the topic of the seminar, it is. Invited to give lectures on the subject, I have been. Take me there, will you?"`
```
```suggestion
			`	"Funny you ask, it is. A xenologist, I am. Alien culture and physiology, I study, and the topic of the seminar, it is. Invited to give lectures on the subject, I have been. Take me there, will you?"`
```

W1zrad commented on data/coalition/coalition missions.txt:
```diff
@@ -1684,3 +1684,180 @@ mission "Coalition: Alpha Encounter"
+				`	"Sorry, I don't have time."`
+						decline
+			label "explanation"
+			`	"Close to Arachi space, humans are, so study radio and TV signals from your part of the galaxy, we are able to. Faint, scattered broadcasts, sophisticated receivers collect, and the signals to us, Arachi colleagues relay. From these broadcasts, we learn. Help us learn more, would you?"`
```
This does raise a question - if the Coalition can study humans in this way, how has humanity not noticed similar signals from the Coalition?

roadrunner56 replied:
According to this https://web.archive.org/web/20160708171434/http://setifaq.org/faq.html#1.2.3, humanity, using current technology, would not be able to detect broadband even a fraction of a light year away from Earth. We can assume that technology in the year 3014 has improved somewhat, but I think that it's plausible that we are still unable to detect broadcasts from the Coalition, whose planets are several light years away from us. The Coalition, because it's been around a lot longer, probably has more sophisticated technology that is able to detect human broadcasts. If anyone more knowledgeable about this can tell me if this reasoning is bull, please do so.

W1zrad replied:
Fair enough. I don't know a whole lot about the subject, so that seems like a reasonable explanation to me.

W1zrad commented on data/coalition/coalition missions.txt:
```diff
@@ -1684,3 +1684,180 @@ mission "Coalition: Alpha Encounter"
+			`	The Arach selects another, this time a Kimek. "All of human space, united politically is it?"`
+			choice
+				`	"Yes, all of human space is united under the Republic."`
+				`	"Yes, but there are internal subdivisions: the Free Worlds in the south, the Republic in the north, and the Syndicate in the east."`
```
This is _technically_ untrue if the player chooses FW Checkmate because that results in them being autonomous from the Republic.

roadrunner56 replied:
To me, 'internal subdivisions' mean 'These divisions have their own system of government and politics'. To be honest with you though, I've never actually played Checkmate.

W1zrad replied:
I think that's fair- feel free to resolve.

W1zrad commented on data/coalition/coalition missions.txt:
```diff
@@ -1684,3 +1684,180 @@ mission "Coalition: Alpha Encounter"
+				`	"No. Technically all of human space is under the Republic, but the Free Worlds and Syndicate have significant autonomy."`
+			`	There are murmurs in the crowd, and several begin to write down notes on various devices."`
+			label "questionthree"
+			`	The Arach picks another Kimek. "Still a problem, piracy is in human space?"`
```
```suggestion
			`	The Arach picks another Kimek. "Still a problem, is piracy in human space?"`
```
Just a little less confusing this way to me.

W1zrad reviewed (changes requested):
Nice mission! I have a few thoughts that didn't fit into any specific suggestions below:
Just something to note: for you labels and branches, you don't _need_ quotation marks around everything as long as there's no spaces. Just something to know if you didn't.
One thing I'd suggest adding is more to the conclusion. Maybe you meet with Aelbris after the seminar and say goodbye? It seems to end pretty abruptly at the moment.
Last, I'd recommend having our resident Coalition expert @Arachi-Lover take a look if he hasn't already. I don't see any of this conflicting what he has planned, but it certainly can't hurt.

Pointedstick commented:
@Arachi-Lover, you up for a review?

Pointedstick reviewed (approved):
All right, merging now then.

thomasballinger commented on data/coalition/coalition missions.txt:
```diff
@@ -1684,3 +1684,184 @@ mission "Coalition: Alpha Encounter"
+					goto donttell
+			label donttell
+			`	"Sorry, I don't know what's there any more than you do."`
+			`	The Saryd walks away, disappointed.`
```
In Discord Xylily points out this seems to be missing a `goto sorry`.

thomasballinger replied:
@roadrunner56 heads up that I posted https://github.com/endless-sky/endless-sky/pull/6211 if that's easiest

roadrunner56 replied:
Thanks for catching that, sorry for not responding until now.

Arachi-Lover commented:
My biggest issue with the mission is how much meta it is, "it's odd you can't tell the Coalition about everyone else in the galaxy, this mission lets you do that." Just don't see it as adding to the game.
 An observation, the Heliarchs are the ones that rule, and are the ones that would, if at all, conduct serious research on other species, yet there is no mention of them throughout the whole mission. 
 In its current state, there's just not really a point for this mission to exist, as I see it, other than to sate a "omg wouldn't it be cool if you could tell X faction about factions A, B, C & D?" idea.
 Another thing is how casually the Drak are mentioned, terrible choice.


On the mission's goal and how it's presented:
- The mission's name is "Transpor Saryd Scientist**s**" however you only have one passenger.
- **"The honor of transporting us to planet, will you give me..."** Us? Again, it is only the one passenger.

On the purpose of this seminar:
- How come xenology is apparently such an important field of the academic world in the Coalition? The three species have been freely living among themselves for 6000 years, and the Heliarchs actively push for cultural exchanges in attempts to neuter chances of seeing those of the other species as "aliens," instead having them seen as fellow citizens. How would xenology even exist in this society? The alien species in question are available for conversation, why would you waste your time studying them like a xenologist instead of conversing with them like a regular person? In fact after so long, with them living together and the government actively pushing for such mingling activities, the cultural differences wouldn't be any more alien than northwest Brazil to the south of the country, where I live. Certainly not something to warrant xenology, so why does this field not only exist but warrant seminars in one of the most prestigious universities in the Coalition?
- **"Close to Arachi space, humans are, so study radio and television signals from your part of the galaxy, we are able to. Faint, scattered broadcasts, sophisticated receivers collect, and the signals to us, Arachi colleagues relay. From these broadcasts, we learn...."** No, they don't learn from that, because radio broadcasts aren't travelling by Jump Drive, and so human ones haven't even gotten close to the halfway point to the nearest Coalition system. Humans have only been in that part of the galaxy for a few centuries, the closest systems are thousands of light years away. So if they can't study humans like that, again what is the point of this? At any rate, anything and everything reliant on these FTL broadcasts is null.


On the seminar questions and the player's answers:
- As it's not made clear at any point: if these scientists are affiliated with the Heliarchs, then they should know about the Hai already, if the player told the Heliarchs, nothing accounts for this, though. If they aren't, and the player told the Heliarchs, that'd imply the Heliarchs have supressed the information, at which point why would they allow the player to say this? Is there not a single Heliarch that connects the dots that First Last is going to the seminar, in spite of their heavy presence in planets and how much they scan ships over and over? The only scenario this doesn't pose a problem is if this is the first instance of the player revealling the Hai to anyone in the Coalition, but nothing in the mission structure accounts for that.
- Both questions about human space are useless if these scientists are affiliated with the Heliarchs, due to the implication of Heliarch contacts with the Poisonwood government from time to time.
- **" 'The Kor Efreti have stayed out of the civil war, living under the Quarg's protection.' At this there are hisses from several in the crowd."** Are these scientists or grade schoolers? If these are Heliarchs, then it's odd, as upon hearing the player say that "The Quarg don't seem oppressive," Heliarchs in RoF explain it from their view, they don't hiss at the player. If they aren't Heliarchs, that'd imply this is a result of propaganda affecting them that much that the mere mention of the Quarg makes them hiss, which is stupid considering these are supposedly scientists.
- **"The last faction, the Korath Exiles, have been banished to the Core by the Quarg, for refusing to give up their ability to manufacture jump drives."** Wrong, it is the Drak/Archons that banished the Korath, that's made very clear during Wanderers.
- **"Escape the cages of the Drak, do any other species?"** Why in the world is an Unfettered here? The Unfettered are the ones with this view, that the Drak keep all in cages, not the Coalition. There's not a mention of the Drak or Archons by anyone in the Coalition so far. This is too casual a name drop, and not at all in an appropriate mission. The Heliarchs hate the Quarg, they have never voiced anything against the Drak, because it wasn't the Drak they fought off. Association with the Quarg doesn't warrant their hatred, for they don't hate humans for being in contact with the Quarg, or in this very mission the mention that the Efreti are protected by the Quarg. The Heliarchs' whole deal is being against the Quarg, not the Drak, don't change that or put in Unfettered views in the mouth of a Coalition character, it makes no sense. If I were to make a mission where the Coalition asks about other species, I would keep the Drak as far from it as possible; ***if*** at all mentioned or a topic, would be the very last thing in the mission and would ***only*** be there in a branch for after the Coalition campaign is completed.
- As a general comment, it feels like First Last is reading from their textbook and trying to answer a teacher during some quiz.


On other things:
 - **"As you exit a restaurant in the spaceport, you notice a rumpled-looking Saryd staring at you. This is not particularly unusual, as consider what a strange sight you must be here in Coalition space. However, he then begins to gallop towards you, hailing you down."** This is incredibly awkward. "You leave X, and see a Saryd staring. Not very odd. Oh, but he's now walking towards you, looks like this is a mission!" I too find it very difficult to make initial encounters with those the player interacts with in missions, and this is about what I'd have for a first draft at that initial bit.
- Why is this character named? There's no need for it, a one-off mission doesn't need a named character, he's of no higher importance than the old Arach with the orphans in one of the missions. If the intent is to reuse the character in later missions, the character in question should be made more interesting.
- On topic of the name, named Coalition characters have their names pulled from combinations of the phrases that make up the ship names. "Aelbris" is not such a combination, and while more possible names are needed (meaning it's okay to have a few names differ from the pattern a bit, provided they'd still be recognizeable as the same "style," as those names could be broken up and added to the pool of the phrases) I recognized this as not being a combination almost immediately. I'm no linguist, but I don't believe that is good, with trying to keep a same style and all. The easiest solution is to just remove the name as it is not needed.
- **"This time, he speaks in a calmer tone of voice."** Odd remark given he had literally only spoken one sentence previously in what could be interpreted as an excited tone (only the "Captain!" bit has the exclamation point, and even then can be just a way of starting a conversation with the player character).
- **"Funny you ask, it is"** Why?
- **"As he enters, he seems fascinated by its structure and design. His gawking makes it difficult to get him to finally settle down into his bunk."** What if I'm in a Coalition ship? Little value in the comment and can easily be made incoeherent if the player just uses one of the easily available local ships at the moment of taking this mission. And whatever happened at the end? Did Captain First Last manage to get this guy to finally settle down into his bunk? Do we find out in the next episode? Quite an abrupt end is what I'm getting at.
- **"During the trip, Aelbris was polite, if a little excited to have a human as his pilot"** Why is this guy fangirling so much? Most Coalition passengers aren't that excited just because their pilot isn't from the Coalition. If you reasoned it to be because of his field of study, should a professional, a scientist, not be more *professional*? How does he expect to conduct proper research if some actual alien shows up? Will he just scream in awe? Just very odd choice of wording to point out his excitement so much.
- **"He asked many questions during the flight"** Such as?
- **"The train you took to the university arrives remarkably quickly..."** There is a railway path up to the peak of a mountain, up to where Starlit University is, the institution placed there with the intent of **"keeping the youth secluded in an ivory tower of learning,"** the building that is **"elevated above the ordinary world of the forest below so that the students can remain detached and focused on their education"**? There is a train that goes there, up to the peak of a mountain?
- **"Now, unplanned, your visit is, so consult with the seminar organizers,..."** Why does this supposed researcher not have something more advanced than face-to-face speech to communicate with?
- **"The honor of having First Last of the Rutilicus system"** When did the player reveal their home system? What if a pilot from another start, when more starts are implemented, takes this mission?
- During certain points the yoda speech doesn't seem to match what else of the Coalition is in. Not versed well enough in english to explain why, might just be an impression, but the style should be checked again.

Pointedstick commented:
Pull requests to fix those issues are welcome :+1:

roadrunner56 commented:
Apologies that I'm only getting to these issues now. 
As to your issue on xenology being a thing in the Coalition, would it be better if I made it clear that this field only studies species outside of the Coalition?
The issue you brought up of human broadcasts reaching the Coalition was discussed in the Discord. I found this article https://web.archive.org/web/20160708171434/http://setifaq.org/faq.html#1.2.3 that states that at our current level of technology, we cannot pick up radio broadcasts from more than a fraction of a light year away. However, my reasoning behind the mission was that the Coalition species have been spacefaring for thousands of years. Presumably, their technology is much better than 21st century human tech, so I reasoned that they could plausibly pick up on TV signals from the Rim.

> Both questions about human space are useless if these scientists are affiliated with the Heliarchs, due to the implication of Heliarch contacts with the Poisonwood government from time to time.

I don't recall seeing this before, could you point me to where it says that? Not doubting you, I'm just genuinely curious as that's a pretty important lore implication that I completely missed out on.

roadrunner56 commented:
Apologies that I'm only getting to these issues now.
As to your issue on xenology being a thing in the Coalition, would it be better if I made it clear that this field only studies species outside of the Coalition?
The issue you brought up of human broadcasts reaching the Coalition was discussed in the Discord. I found this article https://web.archive.org/web/20160708171434/http://setifaq.org/faq.html#1.2.3 that states that at our current level of technology, we cannot pick up radio broadcasts from more than a fraction of a light year away. However, my reasoning behind the mission was that the Coalition species have been spacefaring for thousands of years. Presumably, their technology is much better than 21st century human tech, so I reasoned that they could plausibly pick up on TV signals from the Rim.

> Both questions about human space are useless if these scientists are affiliated with the Heliarchs, due to the implication of Heliarch contacts with the Poisonwood government from time to time.

I don't recall seeing this before, could you point me to where it says that? Not doubting you, I'm just genuinely curious as that's a pretty important lore implication that I completely missed out on.
Everything else I agree with and I'll try making a PR that addresses those issues. Thanks for your critique!

roadrunner56 commented:
As I'm writing the PR, I'm finding it difficult to have such a long mission without once mentioning the Saryd's name. It makes for awkward writing. I'm unfamiliar with Saryd naming conventions, so I would be grateful if you could point me to some resources.

Arachi-Lover commented:
`As to your issue on xenology being a thing in the Coalition, would it be better if I made it clear that this field only studies species outside of the Coalition?`
The Coalition only knew of the Quarg for almost all their time as spacefaring civilizations (not counting one of the species knowing of the other two for obvious reasons), with humans being a recent discovery, and the possibility to reveal the Hai to them. I don't know if people will add more civilizations around the Coalition (as they do say that humans are their most **recent** neighbor, meaning there'd be more, presumably), but humans should still be their most recent "find" so to speak.

So you could make humans themselves the actual focus of it (at least for the time being, mission could be adjusted in the case more species are added), say that humanity being revealed to the Coalition sort of resurrected the field, but the issue with that would be that there's not a way to study humanity without the use of Jump Drives, and the Heliarchs wouldn't supply those for regular visits. What I added with the Alpha Encounter mission has it that Athis is who revealed humanity to the Coalition, and given the implication the Poisonwood governor gives us, it's very likely the Heliarchs keep contact with Poisonwood from time to time, so their information would be what they can get from those sources, moreso Poisonwood as I don't imagine Athis would be very talkative or reliable after centuries of living there, so there needs to be either a mention that these researchers are affiliated with the Heliarchs, or that the Heliarchs share theinformation they get with researchers/the general public (in this case that'd be what information they want to pass on, possibly altered information).

On the broadcasts, my issue isn't that the Coalition can pick them up, it's that the broadcasts could not have arrived at all; humans in ES have been spacefaring for under 1000 years (just over 1000 if we really wanna count the Space Race), and it's an even shorter period that humans have been inhabiting the Rim, while the closest human system to the Coalition is over 1000 lightyears away. The broadcasts simply could not have arrived there yet, unless they're FTL broadcasts or are travelling by the broadcast versions of JDs, which is silly as even if that tech exists, humans and what broadcasts they produce won't make use of said tech.
The hyperspace lanes in human space and in Coalition space are noted as being what allows for the fast communications between systems, but there's no lane connecting the Coalition to human space, so that couldn't be used either. If you go with the initial information from the Alpha + some information from Poisonwood idea, you could delegate the knowledge to those and just do away with the broadcasts, something I feel needs to be done as I don't see a way for the broadcasts to have reached the Coalition without inventing some new technology that I'd say humanity really shouldn't have, given their tech level.

`I don't recall seeing this before, could you point me to where it says that?`
It's from `free worlds start.txt`, line 480: 
			`	She laughs. "You must watch far too many videos. The most powerful species in this galaxy are also the most peace-loving. Trust me, I should know. Here on the edge of human space, we receive... visitors, on occasion. They do not interfere in human events, but they came seeking information on a threat that concerns us both."`
It's a brief mention, and originally, before the Coalition was in the game, thought to refer to the Pug, but since their addition it's generally thought to be Heliarchs visitting the planet, at least from what I remember from discussions.

`As I'm writing the PR, I'm finding it difficult to have such a long mission without once mentioning the Saryd's name. It makes for awkward writing. I'm unfamiliar with Saryd naming conventions, so I would be grateful if you could point me to some resources.`
Yeah it was a nitpick, you can keep the name as is.
The names I've used are just combinations of the words that make up the phrases used as names for Coalition civilian ships, I just eye the list and try to form something that doesn't sound terrible, for some I've varied a bit from the words on the list.
I looked at the list a lot so I recognized yours wasn't a combination from there, but it's best to not have everyone named something you'd see as the name of a Coalition ship, so it's fine to keep the name.

roadrunner56 commented:
Since tracking human broadcasts is out, would it be plausible for the Poisonwood/Republic government to share certain info/documents about humanity to the Heliarchs, and then to have the Heliarchs present some of this info to Coalition researchers?

Arachi-Lover commented:
If they meet with Poisonwood's governor, the governor can give out whatever documents they have access to, maybe not everything the Republic has, given the meetings probably aren't disclosed to the rest of the Republic, so the governor couldn't get more sensitive information, but most forms of documents should be fine, and of course the token Alexandria history-in-a-chip thingy as well for if they wanted more on humanity's history.
The event should be Heliarch controlled, yes, you can choose if answers to some questions have the microphone or whatever device the player speaks into suffer a "malfunction", or the translation device of whoever does the interview suffers such "malfunction," or if they just don't let the researchers ask some questions in the first place, or if they just don't do anything, but are noted to be around, observing, or something else.
You could use a line at the beginning where Aelbris explains his urgency by going "thanks for agreeing to help, I don't wanna be late and miss it, especially since they don't record it" or something like that to make it clear this seminar is very private and selective as to who can participate.

roadrunner56 commented:
Created a PR trying to fix these issues at #6432.
